tools/scripts/xabuild

```diff
@@ -160,6 +160,14 @@ function ConfigureLocalXbuild()
 	done
 }
 
+if [ -z "$CONFIGURATION" ]; then
+		for p in "$*"; do
+				case $p in
+						/p:Configuration=*|-p:Configuration=*) CONFIGURATION="`echo $p | cut -d '=' -f 2`" ;;
```

Comment:
I thought about the "invalid" configurations and considered restricting the check on line 166 to just `Release` and `Debug` but then I thought that if they pass `/p:Configuration=AndroidDebug` then no change to the current behavior will be done, since `CONFIGURATION` will be set to `AndroidDebug` and since such directory doesn't exist at `$prefix`, the behavior will be unchanged wrt to the current one (it will simply continue to look for `Debug` and `Release`)